code/modules/reagents/reagent_containers/chem_pack.dm

```diff
@@ -0,0 +1,53 @@
+/obj/item/reagent_containers/chem_pack
+	name = "intravenous medicine bag"
+	desc = "A plastic pressure bag, or 'chem pack', for IV administration of drugs. It is fitted with a thermosealing strip."
+	icon = 'icons/obj/bloodpack.dmi'
+	icon_state = "chempack"
+	volume = 100
+	reagent_flags = OPENCONTAINER
+	spillable = TRUE
+	obj_flags = UNIQUE_RENAME
+	resistance_flags = ACID_PROOF
+	var/sealed = FALSE
+
+/obj/item/reagent_containers/chem_pack/on_reagent_change(changetype)
+	update_icon()
+
+/obj/item/reagent_containers/chem_pack/update_icon()
+	cut_overlays()
+
+	var/v = min(round(reagents.total_volume / volume * 10), 10)
+	if(v > 0)
+		var/mutable_appearance/filling = mutable_appearance('icons/obj/reagentfillings.dmi', "chempack1")
+		filling.icon_state = "chempack[v]"
+		filling.color = mix_color_from_reagents(reagents.reagent_list)

```

Comment:
usually you just use the master reagent so you can sneak naughty chems in.

Reply:
The idea was that a big brain doctor might be able to guess what certain packs might contain based on the colors. A too strong thializid mix might appear too neon green.

Reply:
I'm not moved either way but I think just seeing an odd color is going to flag knowledgeable people and ruin sabotage potential.